src/actions/projectActions.ts

```diff
@@ -1,25 +1,41 @@
 import request from 'superagent-bluebird-promise';
-import { buildUrl, catchHandler } from './utils/fetch';
+import {Dispatch} from 'redux';
+import {buildUrl, catchHandler} from './utils/fetch';
 import t from '../trans';
-import { busyToggle, success } from './appActions';
-import { ACTION_TYPES } from './utils/ActionTypes';
-import { ProjectModel } from '../components/project/models';
+import {busyToggle, success} from './appActions';
+import {ACTION_TYPES} from './utils/ActionTypes';
+import {ProjectModel, ProjectMonthModel} from '../components/project/models';
 
-export function saveProject(project: ProjectModel, history: any, stayOnPage = false, callback?: (project: ProjectModel) => void) {
-  return dispatch => {
+export function saveProject(project: ProjectModel, history: any) {
+  return (dispatch: Dispatch) => {
     dispatch(busyToggle());
-    return request.post(buildUrl('/projects'))
+    return request
+      .post(buildUrl('/projects'))
       .set('Content-Type', 'application/json')
       .send(project)
       .then(response => {
         dispatch({
           type: ACTION_TYPES.PROJECT_UPDATE,
-          project: response.body
-        })
-        success(t('config.popupMessage'))
-        history.push('/projects')
+          project: response.body,
+        });
+        success(t('config.popupMessage'));
+        history.push('/projects');
       })
       .catch(catchHandler)
       .then(() => dispatch(busyToggle.off()));
   };
 }
+
+export function saveProjectMonth(projectMonth: ProjectMonthModel) {
+  return (dispatch: Dispatch) => {
+    dispatch(busyToggle());
+    return request
+      .post(buildUrl('/projects/month'))
+      .set('Content-Type', 'application/json')
+      .send(projectMonth)
+      .then(response => {
```

Comment:
De store misschien nog updaten met de aangemaakte projectMonths? (of komt dat later?)

Reply:
Zal ik later doen.